lib/sandbox.mli

```diff
@@ -12,3 +12,5 @@ val cmdliner : config Cmdliner.Term.t
 val create : state_dir:string -> config -> t Lwt.t
 (** [create ~state_dir config] is a sandboxing system that keeps state in [state_dir]
     and is configured using [config]. *)
+
+val finished : unit -> unit Lwt.t
```

Comment:
Worth adding comment here describing what `finished` is logically doing in a sandbox?

Reply:
I have added a description.